settings.yml

```diff
@@ -25,7 +25,9 @@ num_columns:
   default: 3
```

Comment:
Let's remove the num_columns settings. Following this change, columns are now fixed-width so this setting is unused.